### PR TITLE
Handle lack of tos gracefully

### LINKED ISSUE #204: Error running django-cloud-deploy-new --use-existing-project
## Issue running command
django-cloud-deploy new

## Environment information

Django Cloud Deploy: [0.1.5]

Platform: [Linux-4.15.0-43-generic-x86_64-with-Ubuntu-18.04-bionic]

Gcloud Version: [233.0.0]

Cloud SQL Proxy Version: [Cloud SQL Proxy: version 1.13; sha ac834ce67862b7c4ed8bdbe86e664572f820c245 built Tue Oct  9 23:39:16 UTC 2018]

Python Version: [3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]]

Docker Version: [Docker version 18.09.0, build 4d60db4]

## Traceback
```.pytb
Traceback (most recent call last):
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/django_cloud_deploy/django_cloud_deploy.py", line 36, in _new
    new.main(args)
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/django_cloud_deploy/cli/new.py", line 213, in main
    backend=args.backend)
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/django_cloud_deploy/workflow/__init__.py", line 214, in create_and_deploy_new_project
    project_id, required_services)
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/django_cloud_deploy/workflow/_enable_service.py", line 53, in enable_required_services
    project_id, service['name'])
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/django_cloud_deploy/cloudlib/enable_service.py", line 51, in enable_service_sync
    response = request.execute()
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/googleapiclient/_helpers.py", line 130, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/home/jdahl/Code/scratch/django-deploy/lib/python3.6/site-packages/googleapiclient/http.py", line 851, in execute
    raise HttpError(resp, content, uri=self.uri)
googleapiclient.errors.HttpError: <HttpError 400 when requesting https://serviceusage.googleapis.com/v1/projects/django-on-gae3/services/compute.googleapis.com:enable?alt=json returned "Operation does not satisfy the following requirements: tos/cloud {The terms of service for this service must be accepted. Please visit the Developers Console to accept the service's terms of service., https://console.developers.google.com/terms/cloud}">

```

### PR DESCRIPTION
**Pull request by enriquejosepadilla**
Fixes #204. Albeit fixes seems to be a strong word. I would say gracefully handles the exception.

### CODE REVIEW

File: django_cloud_deploy/cloudlib/enable_service.py [hunk 1]
```diff
@@ -52,7 +55,17 @@ def enable_service_sync(self, project_id: str, service: str):
...
+        try:
+            response = request.execute()
+        except errors.HttpError as e:
+            if e.resp.status == 400:
+                tos = 'terms of service'
+                if tos in str(e):
+                    url = 'https://console.developers.google.com/terms/cloud'
+                    msg = ('Please accept the terms of service in the Google'
+                           'Cloud Console @ {}'.format(url))
+                    raise crash_handling.UserError(msg)
```
Comment (wangxf123456) [inline django_cloud_deploy/cloudlib/enable_service.py]: Maybe "raise crash_handling.UserError(msg) from e". We are still curious about the original exception.

File: django_cloud_deploy/cloudlib/enable_service.py [hunk 2]
```diff
@@ -52,7 +55,17 @@ def enable_service_sync(self, project_id: str, service: str):
...
+            raise e
```
Comment (wangxf123456) [inline django_cloud_deploy/cloudlib/enable_service.py]:
```python
else:
    raise e
```

This is more clear that we reraise other kinds of exceptions.

File: django_cloud_deploy/cloudlib/enable_service.py (hunk 1 above)
Comment (brianquinlan) [reply django_cloud_deploy/cloudlib/enable_service.py]: What does the message look like if you use `from e` - we want to give the user a simple error message, not a big traceback.

Comment (wangxf123456) [reply django_cloud_deploy/cloudlib/enable_service.py]: I see. @enriquejosepadilla you can ignore my comments here.

**Pull request merged**